src/widgets/StaticCallouts/experiment.js

```diff
@@ -0,0 +1,31 @@
+const PROD_OPTIMIZELY_EXP_ID = '22164741776';
+const STAGE_OPTIMIZELY_EXP_ID = '22174452149';
+
+const activateExperiment = (isEnterpriseUser) => {
+  const optimizely = window.optimizely || null;
+  if (optimizely) {
+    optimizely.push({
+      type: 'user',
+      attributes: {
+        isEnterpriseUser,
+      },
+    });
+    optimizely.push({
+      type: 'page',
+      pageName: 'van_1225_merchandise_2u_lobs_on_learner_home',
+    });
+  }
+};
+
+const getExperimentVariation = () => {
+  const experimentId = process.env.NODE_ENV === 'development' ? STAGE_OPTIMIZELY_EXP_ID : PROD_OPTIMIZELY_EXP_ID;
```

Comment:
Please store the optimizely experiment id in stage and prod configs and use the same name for the variable. 
MERCHANDISE_2U_LOB_EXP_ID
```suggestion
  const experimentId = process.env.MERCHANDISE_2U_LOB_EXP_ID;
```

Reply:
We are using this information only once for this experiment and this is also not sensitive information. I think we can use this here as we are using.